src/functional_test_runner/cli.js

```diff
@@ -28,6 +29,12 @@ if (cmd.verbose) logLevel = 'verbose';
 const log = createToolingLog(logLevel);
 log.pipe(process.stdout);
 
+if (cmd.updateBaselines) {
```

Comment:
Determining whether this test-run should update the snapshots via an environment variable seems rather hacky, is there a way to pass this to the tests via the `config`/`configOverrides` in the `createFunctionalTestRunner`; or a way to run dedicated tests within the apps perhaps by hi-jacking the `grep` extension in mocha?

Reply:
Ah yes, I meant to revisit that.  Found an alternative solution without using environment variables.